l1-contracts/contracts/dev-contracts/test/DummySharedBridge.sol

```diff
@@ -108,14 +109,33 @@ contract DummySharedBridge {
     }
 
     function bridgehubDeposit(
```

Comment:
The `DummySharedBridge` looks like a reimplementation of SharedBridge? Can we use shared bridge or at least inherit shared bridge to DummySharedBridge?

Reply:
I've removed this dummy shared bridge in latest changes, should be fixed.